### DISCUSSION TITLE
Effekseer support

### DISCUSSION THREAD

**Discussion by qoozta** [axmolengine/axmol#943] [General]
The 616cd21 commit added Effekseer libraries, is it tested to work with axmol? I have integrated Effekseer in my fork, there required some changes on both side, the library and the engine.

E.g. in Effekseer+Metal.mm

#include "renderer/backend/metal/Utils.h"
-> #include "renderer/backend/metal/UtilsMTL.h"

bufferM->beginRenderPass(descriptor);
-> auto target = d->getRenderer()->getRenderTarget();
-> bufferM->beginRenderPass(target, descriptor);

**Comment by halx99** [comment]
It was test on windows, will add test for apple(Metal) platform later, thanks for you provide the information.

**Comment by aismann** [comment]
How can I use an Effekseer test?
Is there one (cpp-test)?)

**Reply by halx99** [reply]
will add later

**Reply by halx99** [reply]
> will add later

done

**Comment by aismann** [comment]
Great! Thanks.
Maybe it's better to add 'Effekseer tests' to the 'Extension Tests'?
![image](https://user-images.githubusercontent.com/8652787/199235160-34ebbb25-7ab6-49df-b205-3ef1847d9411.png)

btw
Maybe the 'UITest' also?
![image](https://user-images.githubusercontent.com/8652787/199235795-df401dec-c807-43a2-a9b2-0894b21899aa.png)

'SpineTest' and 'ParticleTest' and all other 'Extensions' test too.
For me it's more consitent.

Or maybe the best:
Add an own (one) project like 'fairgui_test' fors all extensions e.g.: 'Extensions_tests'
